src/core/Effect.cpp

```diff
@@ -52,9 +52,11 @@ Effect::Effect( const Plugin::Descriptor * _desc,
 	m_autoQuitModel( 1.0f, 1.0f, 8000.0f, 100.0f, 1.0f, this, tr( "Decay" ) ),
 	m_autoQuitDisabled( false )
 {
+	m_wetDryModel.setCenterValue( 0 );
+
 	m_srcState[0] = m_srcState[1] = nullptr;
 	reinitSRC();
-	
+
```

Comment:
I mean you should be keeping that tabspace as it helps minimise the diff to one line.

Reply:
> I think it's reasonable to do an on-the-fly fixup like this when it doesn't impact on readability.

Yeah ig. I mentioned it because adherence to style guidelines and not touching existing code if no changes.

Reply:
trailing whitespace.

Reply:
Ok then. Feel free to resolve this one.